pkg/controller/feed/reconcile_test.go

```diff
@@ -49,6 +49,10 @@ var (
 	deletionTime = metav1.Now().Rfc3339Copy()
 )
 
+const (
+	targetDNS = "myservice.mynamespace.svc.cluster.local"
+)
+
 func init() {
 	// Add types to scheme
 	feedsv1alpha1.AddToScheme(scheme.Scheme)
```

Comment:
Remove the `servingv1alpha1.AddToScheme(scheme.Scheme)` line below this one to remove the serving import.